### PR TITLE
Refactor short label

### PR DESCRIPTION
**Pull request by aberonni**
[AB#27477](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/27477)

### CODE REVIEW

File: services/121-service/seed-data/fsp/fsp-africas-talking.json [hunk 1]
```diff
@@ -8,21 +8,19 @@
     {
```
Comment (aberonni) [inline services/121-service/seed-data/fsp/fsp-africas-talking.json:8]: All of the changes within the `services/121-service/seed-data` folder were applied through a custom script (so no manual work was involved).

Comment (RubenGeo) [reply services/121-service/seed-data/fsp/fsp-africas-talking.json:8]: Nice! There are also a lot of formatting changes in those json files. For me these are automatically edited on save if I open a vscode window from the `121-service` folder.

You can also run `npm run test:prettier -- --write` from the root folder. Maybe we should have something that `npm run fix` also runs that command

**Review by RubenGeo — CHANGES_REQUESTED** [review]
Needs more prettier linter magic, to prevent unneeded changes

File: services/121-service/seed-data/fsp/fsp-africas-talking.json (hunk 1 above)
Comment (aberonni) [reply services/121-service/seed-data/fsp/fsp-africas-talking.json:8]: Good that you mention it! These should be picked up automatically, and CI should complain about them if they aren't correct. I thought I address this in https://github.com/global-121/121-platform/pull/5191 but evidently something has slipped through the cracks. Will take a look and report back ✌️

**Comment by aberonni** [comment]
@RubenGeo I pushed the linting fixes. Looks like the CI job is not triggering when it is supposed to. Will address that in a separate PR.

**Review by RubenGeo — APPROVED** [review]
Amazing!

**Pull request merged**